src/pages/staticPages/tools/oncoprinter/Oncoprinter.tsx

```diff
@@ -324,6 +353,18 @@ export default class Oncoprinter extends React.Component<IOncoprinterProps, {}>
                      onMouseEnter={this.onMouseEnter}
                      onMouseLeave={this.onMouseLeave}
                 >
+                    { this.props.store.existCustomDrivers && !this.props.store.customDriverWarningHidden &&
+                        this.props.store.driverAnnotationSettings.customBinary && (
```

Comment:
what does customBinary mean?

Reply:
thats the name we've had for custom binary driver annotations (binary meaning -driver yes, or driver no. as opposed to custom *tiered* annotations)